rosbag2_py/src/rosbag2_py/format_bag_metadata.hpp

```diff
@@ -23,11 +23,22 @@
 namespace rosbag2_py
 {
 
+/**
+ * \brief Available sorting methods for info output.
+ */
+enum class SortingMethod
+{
+  NAME,
+  TYPE,
+  COUNT,
+};
+
```

Comment:
Please rename it to the `InfoSortingMethod` and move it to a separate header file. e.g. `InfoSortingMethod.hpp`
Also please use C++ Doxygen comments style. e.g. 
```cpp
/// \brief Available sorting methods for info output.
```